sql/015-operators-eq.sql

```diff
@@ -0,0 +1,525 @@
+-- Operators for unique comparisons of cs_encrypted_v1 types
+--
+-- Support for the following comparisons:
+--
+--      cs_encrypted_v1 = cs_encrypted_v1
+--      cs_encrypted_v1 <> cs_encrypted_v1
+--      cs_encrypted_v1 = jsonb
+--      cs_encrypted_v1 <> jsonb
+--      cs_encrypted_v1 = text
+--      cs_encrypted_v1 <> text
+--
+
+DROP OPERATOR IF EXISTS = (cs_encrypted_v1, cs_encrypted_v1);
+DROP FUNCTION IF EXISTS cs_encrypted_eq_v1(a cs_encrypted_v1, b cs_encrypted_v1);
+
+CREATE FUNCTION cs_encrypted_eq_v1(a cs_encrypted_v1, b cs_encrypted_v1)
+  RETURNS boolean
+  IMMUTABLE STRICT PARALLEL SAFE
+AS $$
+  DECLARE
+    u boolean;
+    o boolean;
+  BEGIN
+    BEGIN
+      u := (SELECT cs_unique_v1(a) = cs_unique_v1(b));
+    EXCEPTION WHEN OTHERS THEN
+      u := false;
+    END;
+
+    BEGIN
+      o := (SELECT cs_ore_64_8_v1(a) = cs_ore_64_8_v1(b));
```

Comment:
Same for this one.

Reply:
`cs_ore_64_8_v1` is same as unique, pulls the field from the `json`
The returned data is cast as `ore_64_8_v1` and then the underlying ore operators are used for the comparison